pages/linux/toolbox-create.md

```diff
@@ -0,0 +1,16 @@
+# toolbox create
+
+> Create a new `toolbox` container.
+> More information: <https://www.mankier.com/1/toolbox-create>.
+
+- Create a `toolbox` container for a different distribution other than the host:
+
+`toolbox create --distro {{distribution}}`
+
+- Create a `toolbox` container for a different distribution release other than the host:
+
+`toolbox create --release {{release}}`
+
+- Change the name of the image used to create the `toolbox` container:
+
+`toolbox create --image {{name}}`
```

Comment:
So you have to use an image to create a new container? Could you add an example where you specify the image to be used?

Reply:
When creating a container an image would be fetched by default with the distro and release provided. Or else we enter the image name instead of entering the other 2 details. Reference image below: 
![image](https://user-images.githubusercontent.com/26346867/182401535-1da39972-79d7-44ec-bf29-617a54f15167.png)
![image](https://user-images.githubusercontent.com/26346867/182402085-0a5e4a53-9c94-4c0f-b9b1-c6852b3867ad.png)
In my case, the container already exists but you can use this method too.

Reply:
Ah, I think I understand. So it should actually be
```
- Create a `toolbox` container with a custom image:
```
Because `Change the name of the image` sounds a lot like `Rename the image`.

Reply:
Changed it.